build

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -e
+
+####################
+# Set up defaults
+###################
+
+: ${DISTRO:?"Need to set DISTRO"}
+: ${ARCH:?"Need to set ARCH"}
+
+# Defaults for jenkins builds and local builds
+export LOG_LEVEL="debug"
```

Comment:
Thanks for catching that, that was for debug purposes, the switch below was made for that. rebasing this now.